.gitignore

```diff
@@ -1,4 +1,5 @@
 bin
 tmp
 cover.out
-kubeconfig
\ No newline at end of file
+kubeconfig
+.DS_Store
```

Comment:
Could you set up your editor to always add a newline at the end of the file ? If you are using vscode: https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files